static/src/js/routes/Search/Search.jsx

```diff
@@ -128,7 +129,7 @@ export const Search = () => {
         >
           <Routes>
             <Route
-              path="/granules/collection-details"
+              path={routes.COLLECTION_DETAILS.replace(routes.SEARCH, '')}
```

Comment:
I thought about if it was better to define COLLECTION_DETAILS as just /granules/collection-details but, this does seem cleaner so the const in the routes file is accurate to how the app actually resolves the page